### ISSUE TITLE
Display suite test names in output?

### ISSUE THREAD

**Issue by alanshaw** [Gozala/entail#21]
Right now I find it hard to associate log lines with individual tests as there's no delimiter.

Could be a `verbose` option?

**Comment by Gozala** [comment]
Yeah it only prints out test names if there is an error. That said it should not be difficult at all to provide alternative output printer, here is the current implementation

https://github.com/Gozala/subtest/blob/7fc5aec414b75c3b3fc3bfa916fbf4c26b4ed831/src/reporter.js#L15-L90

It basically takes async iterable of updates from the runner

https://github.com/Gozala/subtest/blob/7fc5aec414b75c3b3fc3bfa916fbf4c26b4ed831/src/api.ts#L186-L192

and yields strings that will be printed into a console


You would however need to thread through `verbose` option here

https://github.com/Gozala/subtest/blob/7fc5aec414b75c3b3fc3bfa916fbf4c26b4ed831/src/lib.js#L43-L53


And something similar from the playwright-test although I don't exactly remember how to do that one

https://github.com/Gozala/subtest/blob/7fc5aec414b75c3b3fc3bfa916fbf4c26b4ed831/src/lib.js#L186-L205
